core/src/main/java/jenkins/cli/SafeRestartCommand.java

```diff
@@ -0,0 +1,33 @@
+package jenkins.cli;
```

Comment:
This still needs a license header add either your name or Jenkins project contributors

Reply:
Done!